RELEASE_NOTES.md

```diff
@@ -1,5 +1,220 @@
 
+# WIP_WIP_WIP_WIP Chef Infra Client 16 WIP_WIP_WIP_WIP
+
+## Breaking Changes
+
+### Log Resource Notification Behavior
+
+The log resource in a recipe or resource will no longer trigger notifications by default. This allows authors to more liberally use log resources without impacting the updated resources count or impacting reporting to Automate. This change will impact users that utilized the log resource to aggregate notifications from other resources. If you utilized the log resource to aggregate multiple notifications into a single notification you should convert to using the notify group resource to perform this action.
+
+TODO: Add examples
+
+To restore the previous behavior set `count_log_resource_updates true` in your `client.rb`.
+
+### resource_name / provides behavior change for HWRPs
+
+TODO: https://github.com/chef/chef/pull/9441
+
+### build_essential GCC Updated on Solaris
+
+On Solaris systems we no longer constrain the version of gcc to 4.8.2 in the build_essential resource to allow for gcc 5 installations.
+
+### s390x Packaging
+
+As outlined in our blog post at <https://blog.chef.io/chef-infra-end-of-life-announcement-for-linux-client-on-ibm-s390x-architecture/>, we will no longer be producing s390x platform packages for Chef Infra Client.
+
+### Changes to Knife
+
+#### knife status --long uses cloud attribute
+
+The `knife status --long` resource now uses Ohai's cloud data instead of ec2 specific data. This improves, but changes, the data output for users on non-AWS clouds.
+
+#### knife-acl is now built-in
+
+The knife-acl gem is now part of Chef Infra Client, but many of the commands have been moved into the existing knife user and group namespaces.
+
+TODO: add details on the command changes from https://github.com/chef/chef/pull/9292
+
+#### knife download role/environment format update
+
+The `knife download role` and `knife download environment` commands now include all possible data fields including those without any data set. This new output behavior matches the behavior of other commands such as `knife role show` or `knife environment show`
+
+#### Deprecated knife cookbook site command removed
+
+The previously deprecated `knife cookbook site` commands have been removed. Use the `knife supermarket` commands instead.
+
+#### Deprecated knife data bag create -s short option removed
+
+The deprecated `knife data bag create -s` option that was not properly honored has removed. Use the `--secret` option instead to set a data bag secrete file during data bag creation.
+
+#### sites-cookbooks directory no longer in cookbook_path
+
+The legacy `sites-cookbooks` directory is no longer added the default `cookbook_path` value. With this change any users with a legacy `sites-cookbooks` directory with need to use the `-O` flag to override the cookbook directory when running commands such as `knife cookbook upload`.
```

Comment:
```suggestion
The legacy `sites-cookbooks` directory is no longer added the default `cookbook_path` value. With this change any users with a legacy `sites-cookbooks` directory will need to use the `-O` flag to override the cookbook directory when running commands such as `knife cookbook upload`.
```

Reply:
> The legacy `sites-cookbooks` directory is no longer added the default `cookbook_path` value.

This sentence is missing a word but I don't know enough about it to suggest a proper fix.